tests/helpers.py

```diff
@@ -7,6 +7,8 @@
 PATH_TO_DEFAULT_BINARY = "/tmp/default.out"
 STRIP_ANSI_DEFAULT = True
 DEFAULT_CONTEXT = "-code -stack"
+ARCH = subprocess.check_output("uname --processor".split()).strip().decode("utf-8")
+CI_VALID_ARCHITECTURES = ["x86_64", "i386", "aarch64", "armv7l"]
```

Comment:
let's make this a tuple instead, to prevent accidentally modifying it, which would affect all other tests.